tests/functional/strategies/test_cream_yfi.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import brownie
+
+from brownie import StrategyCreamYFI
+
+STRATEGIES = [StrategyCreamYFI]
+
+
+@pytest.mark.parametrize("Strategy", STRATEGIES)
```

Comment:
The idea here was to parametrize based on multiple strategies, so we'd only have to write the test one time. But you are breaking out each test case into it's own individual file. That means this pattern:
```python
STRATEGIES = [StrategySomething]

@pytest.mark.parametrize("Strategy", STRATEGIES)
def test_strategy_deployment(..., Strategy, ...):
    ...
```
could at least be simplified to this:

```python
def test_strategy_deployment(..., StrategySomething, ...):
    ...
```
I trust that you see why this is the case?
---
Broadly speaking though, it would be nice not to have to break up the cases into individual files to take advantage of fixture isolation, which I think you can do if you use `fn_isolation` instead. Can we give this a shot?

Splitting up the test cases is more broadly destructive, and I wish to avoid that if possible.

Reply:
Yes, regarding the simplification, the only reason I didn't do that was to keep it easy to copy and paste the signatures across for changes. I'll happily adjust it to suit.

Regarding breaking it up, I agree it's horrible for maintenance and readability. I'll attempt to get it working through fn_isolation. I wasn't able to get it to parallelize without setting module_isolation, but I probably just haven't dug deeply enough. I'll do some investigation.

Reply:
@fubuloubu Unfortunately, I do not think fn_isolation will work. I've just reset the code back to it's original state and attempted parallelization with fn_isolation again, and xdist has reverted to only parallelizing modules.

Digging into the [brownie docs](https://eth-brownie.readthedocs.io/en/stable/tests-pytest-intro.html#using-xdist-for-distributed-testing), this is because, "Tests are distributed to workers on a per-module basis.".